doc/release/1.9.2-notes.rst

```diff
@@ -5,15 +5,60 @@ SciPy 1.9.2 Release Notes
 .. contents::
 
 SciPy 1.9.2 is a bug-fix release with no new features
-compared to 1.9.1.
+compared to 1.9.1. It also provides wheel for Python 3.11
+on several platforms.
```

Comment:
last I checked we still have a certificate issue for MacOS python 3.11 ARM and x86, is there something I can backport for that?

Reply:
It should disappear, the fix was merged 3 days ago: https://github.com/actions/python-versions/pull/189